containers/podman-desktop-extension-bootc
# feature: add homepage

feature: add homepage

### What does this PR do?

Adds the following:
* A homepage that lists all the current builds
* Deleting one build
* Deleting multiple builds / bulk deletion
* Be able to click and open the build output directory

### Screenshot / video of UI

https://github.com/containers/podman-desktop-extension-bootc/assets/6422176/3151f45d-2e5a-4eaa-b384-22d7c353705c



### What issues does this PR fix or reference?



Closes https://github.com/containers/podman-desktop-extension-bootc/issues/196

### How to test this PR?



1. Start a build
2. View it in the dashboard
3. Delete the build
4. Try building another build..
5. Click on the folder button (should open in default file manager on
   your system)

Signed-off-by: Charlie Drage <charlie@charliedrage.com>
